tests/components/lutron_caseta/test_diagnostics.py

```diff
@@ -38,151 +38,7 @@ async def test_diagnostics(hass, hass_client) -> None:
-                },
-                "803": {
-                    "device_id": "803",
-                    "current_state": 100,
-                    "fan_speed": None,
-                    "zone": "803",
-                    "name": "Basement Bathroom_Exhaust Fan",
-                    "button_groups": None,
-                    "type": "Switched",
-                    "model": None,
-                    "serial": None,
-                    "tilt": None,
-                    "area": "910",
-                },
-                "804": {
-                    "device_id": "804",
-                    "current_state": 100,
-                    "fan_speed": None,
-                    "zone": "804",
-                    "name": "Master Bedroom_Ceiling Fan",
-                    "button_groups": None,
-                    "type": "FanSpeed",
-                    "model": None,
-                    "serial": None,
-                    "tilt": None,
-                    "area": "1024",
-                },
-                "901": {
-                    "device_id": "901",
-                    "current_state": 100,
-                    "fan_speed": None,
-                    "zone": "901",
-                    "name": "Kitchen_Main Lights",
-                    "button_groups": None,
-                    "type": "WallDimmer",
-                    "model": None,
-                    "serial": 5442321,
-                    "tilt": None,
-                    "area": "1025",
-                },
-                "9": {
-                    "device_id": "9",
-                    "current_state": -1,
-                    "fan_speed": None,
-                    "tilt": None,
-                    "zone": None,
-                    "name": "Dining Room_Pico",
-                    "button_groups": ["4"],
-                    "occupancy_sensors": None,
-                    "type": "Pico3ButtonRaiseLower",
-                    "model": "PJ2-3BRL-GXX-X01",
-                    "serial": 68551522,
-                    "device_name": "Pico",
-                    "area": "1026",
-                },
-                "1355": {
-                    "device_id": "1355",
-                    "current_state": -1,
-                    "fan_speed": None,
-                    "zone": None,
-                    "name": "Hallway_Main Stairs Position 1 Keypad",
-                    "button_groups": ["1363"],
-                    "type": "SunnataKeypad",
-                    "model": "RRST-W3RL-XX",
-                    "serial": 66286451,
-                    "control_station_name": "Main Stairs",
-                    "device_name": "Position 1",
-                    "area": "1205",
-                },
-            },
-            "occupancy_groups": {},
-            "scenes": {},
-        },
-        "entry": {
-            "data": {"ca_certs": "", "certfile": "", "host": "1.1.1.1", "keyfile": ""},
-            "title": "Mock Title",
-        },
-    }
+
```

Comment:
It would still be good to test this even if you need to use a few `ANY` here

Reply:
Can you give me an example of how you would use the ANY?

Reply:
https://github.com/home-assistant/core/blob/16d3cc905f7d9b747d5f1de430dc2a0e9b52c452/tests/components/bluetooth/test_diagnostics.py#L126

Reply:
LOL, that easy?  Ok, no problem.

Reply:
updated